src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import { Suspense, lazy } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { AuthProvider } from '../contexts';
+
+const MainPage = lazy(() => import('./Main'));
+
+const App = () => (
+  <AuthProvider>
```

Comment:
Not sure we need router, suspense, etc. right now.

it seems like authprovider is the only thing we need right now?

Maybe we can delete this file and just add authprovider in index.js, such as
```
<AuthProvider>
   <MainPage />
</AuthProvider>
```